lib/featured-slide.js

```diff
@@ -34,7 +34,8 @@ export default async (initialUrl, targetDir) => {
   const snapshotFilename = `${targetDir}/featured-slide.jpg`;
 
   const url = `http://${host}:${port}/${initialUrl}${getSlideAnchor(featuredSlide.toString())}`;
-
+  
+  console.log("🚀 ~ url:", url)
```

Comment:
I guess this isn't supposed to be part of this PR

Reply:
yes, I should clean this before pushing